docs/anchoring-assets/anchor-platform/sep31/configuration.mdx

```diff
@@ -296,6 +296,8 @@ assets:
     sep38:
       exchangeable_assets:
         - iso4217:BRL
+      country_codes:
+        - BRA
```

Comment:
I don't know if we actually need to have businesses configure country codes for on-chain assets.. can't that be limited to off-chain assets, since thats where the on/off-ramping occurs?

Reply:
Right now if not specified it will give an error. We can look into improving asset configuration separately (so the country code is not required for on-chain assets anymore), WDYT?

Reply:
Sounds good to me.